doc/FAQ.rst

```diff
@@ -0,0 +1,55 @@
+FAQ
+===
+
+Does CastXML work with pure C code too?
+---------------------------------------
+
+Yes and no. CastXML is written as an extension to Clang's C++ parser and therefore
+treats all source files as C++ regardless of their extension.
+If the C code happens to be valid C++ then CastXML will handle it.
+There are a few parts of C not included in C++ that will cause CastXML to produce errors.
+These are documented in Annex C of the C++98 Standard.
```

Comment:
This was true for gccxml but is not for CastXML.  We do support `.c` files in C modes.  Options like `-std=c99` are supported.